HarryJLeigh.FlashCards/FlashCards/Controllers/FlashcardController.cs

```diff
@@ -0,0 +1,68 @@
+using System.Data.SqlClient;
+using Dapper;
+using FlashCards.Data;
+
+namespace FlashCards.Controllers;
+
+public class FlashcardController
+{
+    private DatabaseService DbService { get; }
+    
+    public FlashcardController(DatabaseService dbService) => DbService = dbService;
```

Comment:
🟠 **Code Style**

💡 If you are going to style your constructor like this, you might as well make it a [primary constructor](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/primary-constructors).

Reply:
Newly learned about primary constructors and they're great! Added to the  flashcard, report and stack controllers